### PR TITLE
Various URL bar related design fixes

### LINKED ISSUE #8058: fix radii, colors, focused state styles in URL bar related to new add-funds button
1. box radius should be either 0px or 4px depending on presence of add-funds button

image

2. also change focus-indicator inner shadow radius to match #1.

image

3. Change focus-indicator style from inner shadow to 2px inside border.

image

image

4. RGB value of add-funds button border should match RGB of bookmark button.

image

5. Fix border not being full width. More info here: https://github.com/brave/browser-laptop/pull/7740#pullrequestreview-27529774

### PR DESCRIPTION
**Pull request by NejcZdovc**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Resolves #8058

### Auditors
@bradleyrichter @bsclifton

### Test Plan
- specified in #8058

### CODE REVIEW

File: less/navigationBar.less [hunk 1]
```diff
@@ -660,19 +660,24 @@
...
-  input:focus + legend:before {
+  legend:before {
     content: ' ';
     position: absolute;
     background: @navigationBarBackgroundActive;
-    border-radius: 4px;
-    box-shadow: 0 0 1px @focusUrlbarOutline, inset 0 0 2px @focusUrlbarOutline, inset 0 1px 8px rgba(0, 137, 255, 0.1);
+    border-radius: 0 4px 4px 0;
     color: #333;
+    box-shadow: inset 0 0 0 1px #bbb, inset 0 0 0 3px rgba(55, 169, 253, 0.4);
```
Comment (luixxiul) [inline less/navigationBar.less]: Let's replace `#bbb` and `rgba(55, 169, 253, 0.4)` with variables and add them to variables.less and global.js for future refactoring work.

File: app/renderer/components/publisherToggle.js [hunk 2]
```diff
@@ -132,11 +132,10 @@ const styles = StyleSheet.create({
     WebkitAppRegion: 'no-drag',
     borderWidth: '1px 1px 1px 0px',
     borderStyle: 'solid',
-    borderColor: '#CBCBCB',
+    borderColor: '#BBB',
```
Comment (luixxiul) [inline app/renderer/components/publisherToggle.js]: Let's apply the variable for `box-shadow` of navigationBar.less.

File: less/navigationBar.less (hunk 1 above)
Comment (NejcZdovc) [inline less/navigationBar.less]: Done

File: app/renderer/components/publisherToggle.js (hunk 2 above)
Comment (NejcZdovc) [inline app/renderer/components/publisherToggle.js]: Done

**Comment by NejcZdovc** [comment]
@luixxiul can you please review it again?

**Pull request merged**